java/src/org/openqa/selenium/json/JsonOutput.java

```diff
@@ -271,16 +283,22 @@ public JsonOutput setPrettyPrint(boolean enablePrettyPrinting) {
   }
 
   /**
-   * @param writeClassName
-   * @return
+   * Specify whether the serialized JSON object should include a "class" property whose value is the fully-qualified
+   * class name of the Java object being serialized.
+   *
+   * @param writeClassName {@code false} to omit the "class" property; {@code true} to include this property
```

Comment:
For boolean cases, we typically follow the Google convention of documenting as "whether something or rather". In this case "Whether to include the "class" property"

Reply:
I assume your feedback here refer to the comment for the [writeClassName] parameter. Brevity is the soul of wit. DONE!